src/main/java/ch/njol/skript/registrations/EventValues.java

```diff
@@ -253,10 +253,10 @@ public T get(E event) {
 		for (EventValueInfo<?, ?> ev : eventValues) {
```

Comment:
`ev` and `event` are very similar, can we use `evi` or `info` like in other places?
```suggestion
		for (EventValueInfo<?, ?> evi : eventValues) {
```